dotcom-rendering/cypress/integration/parallel-3/liveblog.interactivity.spec.js

```diff
@@ -0,0 +1,208 @@
+/* eslint-disable mocha/no-exclusive-tests */
+/* eslint-disable mocha/no-pending-tests */
+/* eslint-disable mocha/no-skipped-tests */
+/* eslint-disable no-undef */
+/* eslint-disable func-names */
+import { mockApi } from '../../lib/mocks';
+import { disableCMP } from '../../lib/disableCMP';
+import { setLocalBaseUrl } from '../../lib/setLocalBaseUrl.js';
+import { tweetBlock } from '../../fixtures/manual/tweet-block';
+
+const blogUrl =
+	'https://www.theguardian.com/australia-news/live/2022/feb/22/australia-news-live-updates-scott-morrison-nsw-trains-coronavirus-covid-omicron-weather';
+
+const ignoreTwitterError = () => {
+	cy.on('uncaught:exception', (err) => {
+		if (err.message.includes('Illegal invocation')) {
+			// Thrown by the twitter lib. This error is unrelated to the test in question
+			// so return false to prevent this error from failing this test
+			return false;
+		}
+	});
+};
```

Comment:
Are we sure it's fine to ignore this error?

Could we extract this into a JSDoc style comment to the const?

Reply:
It's conceivable that there is an issue but I'd rather solve problems that get surfaced on the real site, not inside the strange Cypress world where globals like `twttr` can have issues. Plus we're testing that the function works so I'm happy to ignore. 

I've moved the comment into JSDoc 👍